.dotnet/src/Generated/Chat.cs

```diff
@@ -85,32 +79,34 @@ public virtual Result<CreateChatCompletionResponse> CreateChatCompletion(CreateC
         /// </item>
         /// <item>
         /// <description>
-        /// Please try the simpler <see cref="CreateChatCompletionAsync(CreateChatCompletionRequest,CancellationToken)"/> convenience overload with strongly typed models first.
+        /// Please try the simpler <see cref="CreateChatCompletionAsync(CreateChatCompletionRequest)"/> convenience overload with strongly typed models first.
         /// </description>
         /// </item>
         /// </list>
         /// </summary>
         /// <param name="content"> The content to send as the body of the request. </param>
-        /// <param name="context"> The request context, which can override default behaviors of the client pipeline on a per-call basis. </param>
+        /// <param name="options"> The request options, which can override default behaviors of the client pipeline on a per-call basis. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="content"/> is null. </exception>
-        /// <exception cref="MessageFailedException"> Service returned a non-success status code. </exception>
+        /// <exception cref="ClientResultException"> Service returned a non-success status code. </exception>
         /// <returns> The response returned from the service. </returns>
-        public virtual async Task<Result> CreateChatCompletionAsync(RequestBody content, RequestOptions context = null)
+        public virtual async Task<ClientResult> CreateChatCompletionAsync(BinaryContent content, RequestOptions options = null)
```

Comment:
Protocol method takes BinaryContent for input model and RequestOptions; returns ClientResult